exec ${PAGER:-/usr/bin/less} "$0" || exit 1
Executing tests from //glossary/adapters:tests
-----------------------------------------------------------------------------
[36mDiscovery starting.[0m
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[36mDiscovery completed in 493 milliseconds.[0m
[36mRun starting. Expected test count is: 31[0m
[32mHelpersSpec:[0m
[31mprodsec.adapters.organization.HelpersSpec *** ABORTED *** (5 seconds, 323 milliseconds)[0m
[31m  java.lang.IllegalStateException: Could not find a valid Docker environment. Please see logs and check configuration[0m
[31m  at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$getFirstValidStrategy$4(DockerClientProviderStrategy.java:156)[0m
[31m  at java.base/java.util.Optional.orElseThrow(Optional.java:403)[0m
[31m  at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:148)[0m
[31m  at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:146)[0m
[31m  at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:188)[0m
[31m  at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:101)[0m
[31m  at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:107)[0m
[31m  at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:316)[0m
[31m  at com.dimafeng.testcontainers.SingleContainer.start(Container.scala:47)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run(ForAllTestContainer.scala:17)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run$(ForAllTestContainer.scala:13)[0m
[31m  at prodsec.adapters.organization.HelpersSpec.run(HelpersSpec.scala:15)[0m
[31m  at org.scalatest.Suite.callExecuteOnSuite$1(Suite.scala:1175)[0m
[31m  at org.scalatest.Suite.$anonfun$runNestedSuites$1(Suite.scala:1222)[0m
[31m  at scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1328)[0m
[31m  at org.scalatest.Suite.runNestedSuites(Suite.scala:1220)[0m
[31m  at org.scalatest.Suite.runNestedSuites$(Suite.scala:1154)[0m
[31m  at org.scalatest.tools.DiscoverySuite.runNestedSuites(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.Suite.run(Suite.scala:1109)[0m
[31m  at org.scalatest.Suite.run$(Suite.scala:1094)[0m
[31m  at org.scalatest.tools.DiscoverySuite.run(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1322)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1316)[0m
[31m  at scala.collection.immutable.List.foreach(List.scala:333)[0m
[31m  at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1316)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:993)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1482)[0m
[31m  at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.main(Runner.scala:775)[0m
[31m  at org.scalatest.tools.Runner.main(Runner.scala)[0m
[31m  at io.bazel.rulesscala.scala_test.Runner.main(Runner.java:33)[0m
[32mOrganizationUnitToGithubHandleTableSpec:[0m
[31mprodsec.adapters.organization.OrganizationUnitToGithubHandleTableSpec *** ABORTED *** (1 millisecond)[0m
[31m  java.lang.IllegalStateException: Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration[0m
[31m  at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:114)[0m
[31m  at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:146)[0m
[31m  at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:188)[0m
[31m  at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:101)[0m
[31m  at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:107)[0m
[31m  at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:316)[0m
[31m  at com.dimafeng.testcontainers.SingleContainer.start(Container.scala:47)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run(ForAllTestContainer.scala:17)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run$(ForAllTestContainer.scala:13)[0m
[31m  at prodsec.adapters.organization.OrganizationUnitToGithubHandleTableSpec.run(OrganizationUnitToGithubHandleTableSpec.scala:22)[0m
[31m  at org.scalatest.Suite.callExecuteOnSuite$1(Suite.scala:1175)[0m
[31m  at org.scalatest.Suite.$anonfun$runNestedSuites$1(Suite.scala:1222)[0m
[31m  at scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1328)[0m
[31m  at org.scalatest.Suite.runNestedSuites(Suite.scala:1220)[0m
[31m  at org.scalatest.Suite.runNestedSuites$(Suite.scala:1154)[0m
[31m  at org.scalatest.tools.DiscoverySuite.runNestedSuites(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.Suite.run(Suite.scala:1109)[0m
[31m  at org.scalatest.Suite.run$(Suite.scala:1094)[0m
[31m  at org.scalatest.tools.DiscoverySuite.run(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1322)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1316)[0m
[31m  at scala.collection.immutable.List.foreach(List.scala:333)[0m
[31m  at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1316)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:993)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1482)[0m
[31m  at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.main(Runner.scala:775)[0m
[31m  at org.scalatest.tools.Runner.main(Runner.scala)[0m
[31m  at io.bazel.rulesscala.scala_test.Runner.main(Runner.java:33)[0m
[32mOrganizationUnitTableSpec:[0m
[31mprodsec.adapters.organization.OrganizationUnitTableSpec *** ABORTED *** (1 millisecond)[0m
[31m  java.lang.IllegalStateException: Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration[0m
[31m  at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:114)[0m
[31m  at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:146)[0m
[31m  at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:188)[0m
[31m  at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:101)[0m
[31m  at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:107)[0m
[31m  at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:316)[0m
[31m  at com.dimafeng.testcontainers.SingleContainer.start(Container.scala:47)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run(ForAllTestContainer.scala:17)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run$(ForAllTestContainer.scala:13)[0m
[31m  at prodsec.adapters.organization.OrganizationUnitTableSpec.run(OrganizationUnitTableSpec.scala:18)[0m
[31m  at org.scalatest.Suite.callExecuteOnSuite$1(Suite.scala:1175)[0m
[31m  at org.scalatest.Suite.$anonfun$runNestedSuites$1(Suite.scala:1222)[0m
[31m  at scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1328)[0m
[31m  at org.scalatest.Suite.runNestedSuites(Suite.scala:1220)[0m
[31m  at org.scalatest.Suite.runNestedSuites$(Suite.scala:1154)[0m
[31m  at org.scalatest.tools.DiscoverySuite.runNestedSuites(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.Suite.run(Suite.scala:1109)[0m
[31m  at org.scalatest.Suite.run$(Suite.scala:1094)[0m
[31m  at org.scalatest.tools.DiscoverySuite.run(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1322)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1316)[0m
[31m  at scala.collection.immutable.List.foreach(List.scala:333)[0m
[31m  at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1316)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:993)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1482)[0m
[31m  at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.main(Runner.scala:775)[0m
[31m  at org.scalatest.tools.Runner.main(Runner.scala)[0m
[31m  at io.bazel.rulesscala.scala_test.Runner.main(Runner.java:33)[0m
[32mEntityTableSpec:[0m
[31mprodsec.adapters.entity.EntityTableSpec *** ABORTED *** (2 milliseconds)[0m
[31m  java.lang.IllegalStateException: Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration[0m
[31m  at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:114)[0m
[31m  at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:146)[0m
[31m  at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:188)[0m
[31m  at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:101)[0m
[31m  at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:107)[0m
[31m  at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:316)[0m
[31m  at com.dimafeng.testcontainers.SingleContainer.start(Container.scala:47)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run(ForAllTestContainer.scala:17)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run$(ForAllTestContainer.scala:13)[0m
[31m  at prodsec.adapters.entity.EntityTableSpec.run(EntityTableSpec.scala:21)[0m
[31m  at org.scalatest.Suite.callExecuteOnSuite$1(Suite.scala:1175)[0m
[31m  at org.scalatest.Suite.$anonfun$runNestedSuites$1(Suite.scala:1222)[0m
[31m  at scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1328)[0m
[31m  at org.scalatest.Suite.runNestedSuites(Suite.scala:1220)[0m
[31m  at org.scalatest.Suite.runNestedSuites$(Suite.scala:1154)[0m
[31m  at org.scalatest.tools.DiscoverySuite.runNestedSuites(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.Suite.run(Suite.scala:1109)[0m
[31m  at org.scalatest.Suite.run$(Suite.scala:1094)[0m
[31m  at org.scalatest.tools.DiscoverySuite.run(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1322)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1316)[0m
[31m  at scala.collection.immutable.List.foreach(List.scala:333)[0m
[31m  at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1316)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:993)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1482)[0m
[31m  at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.main(Runner.scala:775)[0m
[31m  at org.scalatest.tools.Runner.main(Runner.scala)[0m
[31m  at io.bazel.rulesscala.scala_test.Runner.main(Runner.java:33)[0m
[32mThreatModelFindingTableSpec:[0m
[31mprodsec.adapters.threatModel.ThreatModelFindingTableSpec *** ABORTED *** (1 millisecond)[0m
[31m  java.lang.IllegalStateException: Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration[0m
[31m  at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:114)[0m
[31m  at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:146)[0m
[31m  at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:188)[0m
[31m  at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:101)[0m
[31m  at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:107)[0m
[31m  at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:316)[0m
[31m  at com.dimafeng.testcontainers.SingleContainer.start(Container.scala:47)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run(ForAllTestContainer.scala:17)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run$(ForAllTestContainer.scala:13)[0m
[31m  at prodsec.adapters.threatModel.ThreatModelFindingTableSpec.run(ThreatModelFindingTableSpec.scala:15)[0m
[31m  at org.scalatest.Suite.callExecuteOnSuite$1(Suite.scala:1175)[0m
[31m  at org.scalatest.Suite.$anonfun$runNestedSuites$1(Suite.scala:1222)[0m
[31m  at scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1328)[0m
[31m  at org.scalatest.Suite.runNestedSuites(Suite.scala:1220)[0m
[31m  at org.scalatest.Suite.runNestedSuites$(Suite.scala:1154)[0m
[31m  at org.scalatest.tools.DiscoverySuite.runNestedSuites(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.Suite.run(Suite.scala:1109)[0m
[31m  at org.scalatest.Suite.run$(Suite.scala:1094)[0m
[31m  at org.scalatest.tools.DiscoverySuite.run(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1322)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1316)[0m
[31m  at scala.collection.immutable.List.foreach(List.scala:333)[0m
[31m  at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1316)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:993)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1482)[0m
[31m  at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.main(Runner.scala:775)[0m
[31m  at org.scalatest.tools.Runner.main(Runner.scala)[0m
[31m  at io.bazel.rulesscala.scala_test.Runner.main(Runner.java:33)[0m
[32mAssetsOwnerTableSpec:[0m
[31mprodsec.adapters.assetsDatabase.AssetsOwnerTableSpec *** ABORTED *** (2 milliseconds)[0m
[31m  java.lang.IllegalStateException: Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration[0m
[31m  at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:114)[0m
[31m  at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:146)[0m
[31m  at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:188)[0m
[31m  at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:101)[0m
[31m  at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:107)[0m
[31m  at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:316)[0m
[31m  at com.dimafeng.testcontainers.SingleContainer.start(Container.scala:47)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run(ForAllTestContainer.scala:17)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run$(ForAllTestContainer.scala:13)[0m
[31m  at prodsec.adapters.assetsDatabase.AssetsOwnerTableSpec.run(AssetsOwnerTableSpec.scala:17)[0m
[31m  at org.scalatest.Suite.callExecuteOnSuite$1(Suite.scala:1175)[0m
[31m  at org.scalatest.Suite.$anonfun$runNestedSuites$1(Suite.scala:1222)[0m
[31m  at scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1328)[0m
[31m  at org.scalatest.Suite.runNestedSuites(Suite.scala:1220)[0m
[31m  at org.scalatest.Suite.runNestedSuites$(Suite.scala:1154)[0m
[31m  at org.scalatest.tools.DiscoverySuite.runNestedSuites(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.Suite.run(Suite.scala:1109)[0m
[31m  at org.scalatest.Suite.run$(Suite.scala:1094)[0m
[31m  at org.scalatest.tools.DiscoverySuite.run(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1322)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1316)[0m
[31m  at scala.collection.immutable.List.foreach(List.scala:333)[0m
[31m  at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1316)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:993)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1482)[0m
[31m  at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.main(Runner.scala:775)[0m
[31m  at org.scalatest.tools.Runner.main(Runner.scala)[0m
[31m  at io.bazel.rulesscala.scala_test.Runner.main(Runner.java:33)[0m
[32mFindingDBFetcherSpec:[0m
[31mprodsec.adapters.findingDatabase.FindingDBFetcherSpec *** ABORTED *** (2 milliseconds)[0m
[31m  java.lang.IllegalStateException: Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration[0m
[31m  at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:114)[0m
[31m  at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:146)[0m
[31m  at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:188)[0m
[31m  at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:101)[0m
[31m  at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:107)[0m
[31m  at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:316)[0m
[31m  at com.dimafeng.testcontainers.SingleContainer.start(Container.scala:47)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run(ForAllTestContainer.scala:17)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run$(ForAllTestContainer.scala:13)[0m
[31m  at prodsec.adapters.findingDatabase.FindingDBFetcherSpec.run(spec.scala:27)[0m
[31m  at org.scalatest.Suite.callExecuteOnSuite$1(Suite.scala:1175)[0m
[31m  at org.scalatest.Suite.$anonfun$runNestedSuites$1(Suite.scala:1222)[0m
[31m  at scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1328)[0m
[31m  at org.scalatest.Suite.runNestedSuites(Suite.scala:1220)[0m
[31m  at org.scalatest.Suite.runNestedSuites$(Suite.scala:1154)[0m
[31m  at org.scalatest.tools.DiscoverySuite.runNestedSuites(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.Suite.run(Suite.scala:1109)[0m
[31m  at org.scalatest.Suite.run$(Suite.scala:1094)[0m
[31m  at org.scalatest.tools.DiscoverySuite.run(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1322)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1316)[0m
[31m  at scala.collection.immutable.List.foreach(List.scala:333)[0m
[31m  at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1316)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:993)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1482)[0m
[31m  at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.main(Runner.scala:775)[0m
[31m  at org.scalatest.tools.Runner.main(Runner.scala)[0m
[31m  at io.bazel.rulesscala.scala_test.Runner.main(Runner.java:33)[0m
[32mEmployeeTableSpec:[0m
[31mprodsec.adapters.employees.EmployeeTableSpec *** ABORTED *** (1 millisecond)[0m
[31m  java.lang.IllegalStateException: Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration[0m
[31m  at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:114)[0m
[31m  at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:146)[0m
[31m  at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:188)[0m
[31m  at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:101)[0m
[31m  at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:107)[0m
[31m  at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:316)[0m
[31m  at com.dimafeng.testcontainers.SingleContainer.start(Container.scala:47)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run(ForAllTestContainer.scala:17)[0m
[31m  at com.dimafeng.testcontainers.ForAllTestContainer.run$(ForAllTestContainer.scala:13)[0m
[31m  at prodsec.adapters.employees.EmployeeTableSpec.run(EmployeeTableSpec.scala:15)[0m
[31m  at org.scalatest.Suite.callExecuteOnSuite$1(Suite.scala:1175)[0m
[31m  at org.scalatest.Suite.$anonfun$runNestedSuites$1(Suite.scala:1222)[0m
[31m  at scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1328)[0m
[31m  at org.scalatest.Suite.runNestedSuites(Suite.scala:1220)[0m
[31m  at org.scalatest.Suite.runNestedSuites$(Suite.scala:1154)[0m
[31m  at org.scalatest.tools.DiscoverySuite.runNestedSuites(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.Suite.run(Suite.scala:1109)[0m
[31m  at org.scalatest.Suite.run$(Suite.scala:1094)[0m
[31m  at org.scalatest.tools.DiscoverySuite.run(DiscoverySuite.scala:30)[0m
[31m  at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1322)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1316)[0m
[31m  at scala.collection.immutable.List.foreach(List.scala:333)[0m
[31m  at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1316)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:993)[0m
[31m  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1482)[0m
[31m  at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:971)[0m
[31m  at org.scalatest.tools.Runner$.main(Runner.scala:775)[0m
[31m  at org.scalatest.tools.Runner.main(Runner.scala)[0m
[31m  at io.bazel.rulesscala.scala_test.Runner.main(Runner.java:33)[0m
[36mRun completed in 5 seconds, 860 milliseconds.[0m
[36mTotal number of tests run: 0[0m
[36mSuites: completed 1, aborted 8[0m
[36mTests: succeeded 0, failed 0, canceled 0, ignored 0, pending 0[0m
[31m*** 8 SUITES ABORTED ***[0m
